core/app/models/spree/return_authorization.rb

```diff
@@ -8,6 +8,7 @@ class ReturnAuthorization < Spree::Base
 
     belongs_to :stock_location
     belongs_to :reason, class_name: 'Spree::ReturnReason', foreign_key: :return_reason_id
+    belongs_to :recipient, class_name: Spree.user_class.to_s
```

Comment:
Cool didn't know that existed. Will update.

As a side note, we have the same `Spree.user_class` in our migration file. Is there a better way around using Spree.user_class there?